Classes/Settings.cs

```diff
@@ -369,6 +330,36 @@ internal static int RssServerNumRecentEps
             }
         }
 
+        internal static string SetSaveFolder(Model.Programme progInfo)
```

Comment:
I'd prefer these not to have `Get*` and `Set*` prefixes - e.g. just use the original names and overloading will take care of the rest.